src/tokens/AssetItem.tsx

```diff
@@ -42,7 +43,7 @@ export const PositionItem = ({ position }: { position: Position }) => {
       onPress={onPress}
     >
       <View style={styles.row}>
-        <Image source={{ uri: position.displayProps.imageUrl }} style={styles.tokenImg} />
+        <TokenIcon token={position} viewStyle={styles.tokenImgContainer} />
```

Comment:
The new asset icon with the logo is applicable only to the new Assets page, we want to leave the existing Asset list and Dapp positions as is.